recipe/meta.yaml

```diff
@@ -22,7 +22,6 @@ source:
 
 build:
   number: {{ build_number }}
-  noarch: python
```

Comment:
This was here to make sure we don't rebuild for every Python version as the packages using Python are `noarch`

The lint here appears to be mistaken: https://github.com/conda-forge/conda-smithy/issues/1887#issuecomment-2244323850